NetKAN/BD-AWP.netkan

```diff
@@ -0,0 +1,23 @@
+{
+    "$kref": "#/ckan/kerbalstuff/1366",
+    "spec_version": "v1.4",
+    "license": "MIT",
+    "identifier": "BD-AWP",
+    "install": [
+        {
+            "find": "AWP",
+            "install_to": "GameData"
+        },
+        {
+            "file": "INSTALLATION GUIDE.txt",
```

Comment:
We can safely exclude the installation of this file. Having it in the install stanza leaves it vulnerable to breaking if the author moves it or renames it.